.gitignore

```diff
@@ -0,0 +1,104 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
add swap files to gitignore, remove other unnecessary stuff.

Reply:
added default ones.Will change